content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review.md

```diff
@@ -16,9 +16,11 @@ topics:
   - Pull requests
 shortTitle: Request a PR review
 ---
-Owners and collaborators on a repository owned by a user account can assign pull request reviews. Organization members with triage permissions to a repository can assign a pull request review.
+Repositories belong to a user account (a single individual owner) or an organization account (a shared account with numerous collaborators or maintainers). You can read more about the different account types [here](/get-started/learning-about-github/types-of-github-accounts). Owners and collaborators on a repository owned by a user account can assign pull request reviews. Likewise, organization members with triage permissions can also assign a pull request. 
 
-Owners or collaborators can assign a pull request review to any person that has been explicitly granted [read access](/articles/access-permissions-on-github) to a user-owned repository. Organization members can assign a pull request review to any person or team with read access to a repository. The requested reviewer or team will receive a notification that you asked them to review the pull request. {% ifversion fpt or ghae or ghes or ghec %}If you request a review from a team and code review assignment is enabled, specific members will be requested and the team will be removed as a reviewer. For more information, see "[Managing code review settings for your team](/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team)."{% endif %}
+In order to request a pull request review, you will need [write access](/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization) permissions. Conversely, owners or collaborators can assign a pull request review to any person that has been granted [read access](/get-started/quickstart/github-glossary#read-access) only.
+
+Organization members can also assign a pull request review to any person or team with read access to a repository. The requested reviewer or team will receive a notification that you asked them to review the pull request. {% ifversion fpt or ghae or ghes or ghec %}If you request a review from a team and code review assignment is enabled, specific members will be requested and the team will be removed as a reviewer. For more information, see "[Managing code review settings for your team](/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team)."{% endif %}
```

Comment:
Maybe, to reinforce the point, we should add that member must have write access:
```suggestion
Organization members with write access can also assign a pull request review to any person or team with read access to a repository. The requested reviewer or team will receive a notification that you asked them to review the pull request. {% ifversion fpt or ghae or ghes or ghec %}If you request a review from a team and code review assignment is enabled, specific members will be requested and the team will be removed as a reviewer. For more information, see "[Managing code review settings for your team](/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team)."{% endif %}
```

Reply:
I agree with this, too. Thank you for taking the time to make such great notes. I will work on all this for future issues.